charts/prometheus-kafka-exporter/Chart.yaml

```diff
@@ -1,9 +1,9 @@
-apiVersion: v1
+apiVersion: v2
 appVersion: "v1.2.0"
 description: A Helm chart to export the metrics from Kafka in Prometheus format using the kafka-exporter from https://github.com/danielqsj/kafka_exporter
 name: prometheus-kafka-exporter
 home: https://github.com/danielqsj/kafka_exporter
-version: 0.2.0
+version: 0.2.1
```

Comment:
As helm v2 api is used this is a breaking change and new chart version should be 1.0